src/components/DynamicFormContainer/DynamicForm/DynamicForm.tsx

```diff
@@ -27,6 +28,7 @@ export interface DynamicFormProps {
 
 export const DynamicFormRaw: FunctionComponent<DynamicFormProps> = ({
   schema,
+  uiSchema = {},
```

Comment:
Is there a need to assign a default to this? Asking because it could alter the behavior of the component if it expects a pass-through.